cllm-website/docs/getting-started.md

```diff
@@ -0,0 +1,24 @@
+---
+sidebar_position: 2
+---
+
+# Getting Started
+
+Welcome to the **CLLM** documentation! This guide will help you get started with **CLLM** and show you how to use it to improve your CLI experience.
```

Comment:
How about add short description about cllm?

Reply:
`Out Docs` has already short description about cllm in introduction tab, I think this is not necessary in `Getting Started` tab.
I want to know your opinion